packages/core/src/divide/halfUp.ts

```diff
@@ -12,12 +12,20 @@ export const halfUp: DivideOperation = (amount, factor, calculator) => {
   const remainder = absoluteFn(calculator.modulo(amount, factor));
   const difference = calculator.subtract(factor, remainder);
   const isLessThanHalf = greaterThanFn(difference, remainder);
-  const isPositive = greaterThanFn(amount, calculator.increment(zero));
+  const isPositive = greaterThanFn(amount, zero);
+
+  // Round up in these three cases:
+  // (1) the quotient is half:
+  //   e.g. ... -2.5,  -1.5,  -0.5,  0.5,  1.5,  2.5 ...
+  // (2) the quotient is positive, and greater than half:
+  //   e.g.                          0.51, 1.51, 2.51 ...
+  // (3) the quotient is negative, and less than half:
+  //   e.g. ... -2.49, -1.49, -0.49
 
   if (
     isHalfFn(amount, factor) ||
-    (isLessThanHalf && !isPositive) ||
-    (!isLessThanHalf && isPositive)
+    (isPositive && !isLessThanHalf) ||
+    (!isPositive && isLessThanHalf)
```

Comment:
Why are these flipped? All the tests pass if I leave them as they were.

Reply:
To be consistent with preceding comment explaining what I'm doing.

I also reckon humans handle positive numbers better, so this can be viewed as: (1) addressing the problem, then (2) the common case then (3) the less common case.

I don't feel strongly about this, except that the comment should be amended if this change is reverted.

Reply:
That makes sense to me, I was just checking that there was a reason. It wasn't apparent.